controllers/user_controller.go

```diff
@@ -0,0 +1,59 @@
+package controllers
+
+import (
+	"net/http"
+
+	"github.com/drift-org/backend/models"
+	"github.com/gin-gonic/gin"
+	"github.com/kamva/mgm/v3"
+)
+
+type UserController interface {
+	Get(context *gin.Context)
+}
+
+type userController struct {
+}
+
+func NewUserController() UserController {
+	return &userController{}
+}
+
+func (ctrl *userController) Get(context *gin.Context) {
+	// Id refers to MongoDB user object ID. Specificty refers to the verbocity of the returned friends array
```

Comment:
nit: typo in Specificity 😬 (and while youre at it would be nice to make Id --> ID for consistency)

i feel cruel with these comment-nits, sorry aiden 😔